interface/code_systems/list_staged.php

```diff
@@ -205,9 +205,20 @@
                     array_push($revisions, $temp_date);
                     $supported_file = 1;
                 } elseif (preg_match("/SnomedCT_SpanishRelease-es_PRODUCTION_([0-9]{8})[0-9a-zA-Z]{8}.zip/", $file, $matches)) {
-                    // Hard code the version SNOMED feed to be International:Spanish
+                    // Hard code the version SNOMED feed to be International:Spanishlang
                     //
-                    $version = "International:Spanish";
+                    $version = "International:Spanishlang";
```

Comment:
$version = "International:SpanishLang";

Reply:
Yes, it's better there. The idea is that all Spanish-speaking countries use the same values: International:SpanishLang (Spain, Argentina, Uruguay, etc...). From now on, that country that needs it must add an entry only in openemr/interfase/code_system/list_staged.php with the name of the compressed file, which is specific to each country.

Reply:
I added only Argentina, because I don't know what file name format the other countries use.

Reply:
ok, just have to correct that lowecase `l` there